Cargo.toml

```diff
@@ -97,6 +97,9 @@ optional = true
 [dependencies.fail]
 git = "https://github.com/pingcap/fail-rs.git"
 
+[replace]
+"protobuf:1.4.1" = { git = "https://github.com/busyjay/rust-protobuf.git", branch = "set-recurse-limit" }
```

Comment:
do we need to update kvproto?

Reply:
cargo will handle this automatically for the configuration in replace section.